seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -346,10 +346,10 @@
         "We can also divide one number by another.",
         "JavaScript uses the <code>/</code> symbol for division.",
         "<h4>Instructions</h4>",
-        "Change the <code>0</code> so that quotient will equal <code>2</code>."
+        "Change the <code>0</code> so that <code>answer</code> will equal <code>2</code>."
```

Comment:
I think the variable name `answer` should be changed to `division` because in the challenge "Waypoint: Multiply Two Decimals with JavaScript", the variable is called as `product`.